README.md

```diff
@@ -294,7 +295,29 @@ For **∅ or more** and **1 or more** element(s) of the same type you can use th
 Union types are denoted with the pipe symbol, `|`:
 
 ```js
-(userInput: String|Number) => String|Number
+(userInput: String | Number) => String | Number
+```
+
+### Negated Types
+
+It is sometime easier and more informative to delimit a type by defining what it's not. The negation operator let's you exclude by substracting from `Any`.
+
+```js
+JSON::parse(String, reviver: Function)
+  => Boolean | Number | String | Object | Array | null,
+  throws SyntaxError
+
+// is less concise than
+
+JSON::parse(String, reviver: Function)
+  => !Function & !Void & !Symbol,
+  throws SyntaxError
+
+// which is equivalent to
```

Comment:
This example is not equivalent. `&` indicates a single value which satisfies the interface for all of the specified types, where as the above example would reject any value that satisfies the `Symbol` type, regardless of whether or not it also satisfied `Void` and `Function`.

The example below can't be satisfied by any type, because nothing can be simultaneously satisfy `Void` and `Symbol` and `Function` (if a Void return was evaluated, it would be `undefined`).

Reply:
This would hold if you swapped out the `&` for `|` in the second example.

Reply:
_is not Function_ *and* _is not Void_ *and* _is not Symbol_ (it's an intersection > it must be `&`)
equals
_is Boolean_ *or* _is Number_ *or* _is String_ *or* _is Object_ *or* _is Array_ *or* _is null_

With 2 caveats:

1. *is Object* may have different interpretations
2. it's highly unlikely but a new builtin might be introduced (e.g. Symbol was)

Reply:
That is correct for the first example: `!Function & !Void & !Symbol` - an intersection of the negated types.

However, the second example: `!(Function & Void & Symbol)` does not mean the same thing.

That one means is not (Function and Void and Symbol) -- where `Function & Void & Symbol` are an intersection of the non-negated types, which means to be rejected by the second example, a value would have to satisfy the interface of all three, which is impossible.